### PR TITLE
Updated pbar description: custom desc + [epoch/n_epochs]

### PR DESCRIPTION
**Pull request by vfdev-5**
Description:
Minor pbar update to display current epoch / num_epochs with custom description:
```python
    pbar = ProgressBar(desc="Validation")
    pbar.attach(engine, output_transform=lambda x: x)
    engine.run(loader, max_epochs=n_epochs)
    expected = u'Validation [10/10]: [4/5]  80%|████████  , output=1.00e+00 [00:00<00:00]'
```

Check list:
* [x] Tests updated

cc @amitibo @miguelvr

### CODE REVIEW

**Comment by amitibo** [comment]
@vfdev-5 I think you can write:
```
desc = "Epoch" if 'desc' not in self.tqdm_kwargs else self.tqdm_kwargs['desc']
```
easier this way:
```
desc = self.tqdm_kwargs.get("desc", "Epoch")
```

If I am correct this PR forces the description to include the [epoch/max_epoch] construct, whether the user wants it or not. Is this what you want? If so, maybe it is better to describe this in the docstring.

**Comment by vfdev-5** [comment]
@amitibo yes, you are correct. Will fix the code and the docstring.

**Comment by vfdev-5** [comment]
@amitibo updated docs and the code such that append to desc `[curr_epoch/num_epoch]` if more than one.

File: ignite/contrib/handlers/tqdm_logger.py [hunk 1]
```diff
@@ -17,14 +17,12 @@ class ProgressBar(object):
...
-              percentage, rate, rate_fmt, rate_noinv, rate_noinv_fmt,
-              rate_inv, rate_inv_fmt, elapsed, remaining, desc, postfix.
-            Note that a trailing ": " is automatically removed after {desc}
-            if the latter is empty.
-        **tqdm_kwargs: kwargs passed to tqdm progress bar
+            r_bar='| {n_fmt}/{total_fmt} [{elapsed}<{remaining}, {rate_fmt}{postfix}]'. For more details on the
+            formatting, see `tqdm docs <https://tqdm.github.io/docs/tqdm/>`_.
+        **tqdm_kwargs: kwargs passed to tqdm progress bar.
+            By default, progress bar description displays "Epoch [5/10]". If tqdm_kwargs defines
+            `desc`, e.g. "Predictions", than the description is "Predictions [5/7]" if number of epochs is more
```
Comment (jasonkriss) [inline ignite/contrib/handlers/tqdm_logger.py]: can we change this to "Predictions [5/10]"? Don't want anyone getting confused over why it went from "5/10" to "5/7"

Comment (vfdev-5) [reply ignite/contrib/handlers/tqdm_logger.py]: Yes, you are right. It is unclear what are these numbers 5, 7 and 10. I'll improve this.

Comment (jasonkriss) [reply ignite/contrib/handlers/tqdm_logger.py]: @vfdev-5 looks perfect now! I'll let you merge when you are ready.

**Pull request merged**